resources/assets/coffee/react/_components/beatmapset-panel.coffee

```diff
@@ -102,9 +102,12 @@ class @BeatmapsetPanel extends React.PureComponent
             src: beatmapset.covers.card
           div className: 'beatmapset-panel__image-overlay'
           div className: 'beatmapset-panel__status-container',
-            if beatmapset.video or beatmapset.storyboard
+            if beatmapset.video
               div className: 'beatmapset-panel__video-icon',
                 i className: 'fas fa-film fa-fw'
+            if beatmapset.storyboard
+              div className: 'beatmapset-panel__storyboard-icon',
```

Comment:
This class doesn't exist and thus not properly styled. If anything this should use same class as video (renamed, perhaps).

Reply:
`__extra-icon`?

![](https://puu.sh/DaFmL/337dd7d35c.png)

Reply:
...yes